Makefile

```diff
@@ -202,10 +157,33 @@ version: VERSION mods/ra/mod.yaml mods/cnc/mod.yaml mods/d2k/mod.yaml mods/ts/mo
 		rm $${i}.tmp; \
 	done
 
-install: dependencies core install-core
+install: core install-core
```

Comment:
If we make `install` depend on `install-dependencies` then the Snap package wouldn't need to add an extra `make` call in the build script immediately this code is merged.